scripts/prepare_sector_network.py

```diff
@@ -1090,6 +1090,16 @@ def prepare_costs(cost_file, params, nyears):
 
     costs = costs.fillna(params["costs"]["fill_values"])
 
+    for attr in ("investment", "lifetime", "FOM", "VOM", "efficiency", "fuel"):
+        overwrites = params["costs"].get(attr)
+        if overwrites is not None:
+            breakpoint()
```

Comment:
Let's leave this on a shelf for upstream. But I think it would be a valuable OS contribution to find a good solution that is less confusing and that we can contribute upstream down the line

Reply:
Sure ! We can open an issue upstream.